extensions/README.md

```diff
@@ -47,9 +47,10 @@ An extension can add new fields to STAC entities (content extension), or can add
 
 | Extension Name (Prefix)                                      | Scope            | Description                                                  | Maturity |
 | ------------------------------------------------------------ | ---------------- | ------------------------------------------------------------ | -------- |
+| [Asset Definition](asset/README.md) (`asset`)                  | Collection | Provides a way to specify details about what assets may be found in Items belonging to a collection | *Proposal* |
```

Comment:
The extension doesn't specify a prefix in the documentation, but here one is specified? Should probably be (-) or so.